src/collection.js

```diff
@@ -3,25 +3,23 @@
 import BaseAdapter from "./adapters/base";
 import { waterfall, deepEqual } from "./utils";
 import { v4 as uuid4 } from "uuid";
-import { isUUID, pFinally } from "./utils";
+import { isUUID, pFinally, omitKeys } from "./utils";
 
-const RECORD_FIELDS_TO_CLEAN = ["_status", "last_modified"];
+const RECORD_FIELDS_TO_CLEAN = ["_status"];
 const AVAILABLE_HOOKS = ["incoming-changes"];
 
+
 /**
- * Cleans a record object, excluding passed keys.
- *
- * @param  {Object} record        The record object.
- * @param  {Array}  excludeFields The list of keys to exclude.
- * @return {Object}               A clean copy of source record object.
+ * Compare two records omitting local fields and synchronization
+ * attributes (like _status and last_modified)
+ * @param {Object} a    A record to compare.
+ * @param {Object} b    A record to compare.
+ * @return {boolean}
  */
-export function cleanRecord(record, excludeFields=RECORD_FIELDS_TO_CLEAN) {
```

Comment:
That's a major change, with no prior notice...FWIW it's used [in kinto-admin](https://github.com/Kinto/kinto-admin/blob/master/scripts/reducers/conflicts.js#L1).